### PR TITLE
Add Toxic Sewers, Mad Lab, and The Hive codes to pcstats.js

### PR DESCRIPTION
**Pull request by WesTyler**

### CODE REVIEW

File: lib/pcstats.js [hunk 1]
```diff
@@ -34,9 +35,10 @@ var shortdungeonnames = { // sorted by "tier"
 	18: 'Sprite',
 	24: 'Manor',
 	14: 'UDL',
+	42: 'Sewer',
 	15: 'Abyss',
 	22: 'Trench',
-	21: 'Tomb',
+	21: 'Tomb'
```
Comment (Nightfirecat) [inline lib/pcstats.js]: No need to remove the trailing comma.

**Comment by Nightfirecat** [comment]
Add the trailing commas back please?

Otherwise, LGTM. :+1:

File: lib/pcstats.js (hunk 1 above)
Comment (WesTyler) [inline lib/pcstats.js]: Oh, haha whoops. My linter strikes again. I'll fix that real quick, thanks.

File: lib/pcstats.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ var pcstatnames = {
 	22: 'TrenchesCompleted',
 	23: 'JunglesCompleted',
 	24: 'ManorsCompleted',
+	42: 'SewersCompleted'
```
Comment (Nightfirecat) [inline lib/pcstats.js]: This one too!

**Comment by WesTyler** [comment]
While we're here, do you know of any source of truth on these "id"s? I just happened to notice this was the Sewers code after running a couple of them. I dug around a bit and couldn't find where these are coming from, or I would add some more in.

File: lib/pcstats.js (hunk 1 above)
Comment (Nightfirecat) [inline lib/pcstats.js]: No worries. Thanks for being so responsive. :)

**Comment by Nightfirecat** [comment]
I can honestly say I don't have the faintest idea. To my knowledge, most of the "inner workings" of Muledump come from getting copies of server content/config, which I'm not fully aware how it's obtained. I know it's fairly regularly scraped and hosted here if you want to go digging: http://static.drips.pw/rotmg/

**Comment by tischepe** [comment]
I don't think Sewer should be added to shortdungeonnames since as far as I know it is not necessary for the tunnel rat fame bonus.
Maybe shortdungeonname should b renamed to reflect its purpose.

**Comment by tischepe** [comment]
@Nightfirecat An older/partial version of the code used to generate http://static.drips.pw/rotmg/ can be found here: https://gist.github.com/atomizer/6705319

**Comment by WesTyler** [comment]
> Maybe shortdungeonname should be renamed to reflect its purpose.

Yeah, I didn't know what that was used for. I'll remove that from this PR :)

**Comment by WesTyler** [comment]
@atomizer are you still maintaining this project, or should somebody fork?

**Pull request closed without merging**